### ISSUE TITLE
evaluating speed is so slow

### ISSUE THREAD

**Issue by kaixinxiao** [junfu1115/DANet#56]
When I use test.py to evaluate model, the speed is so slow. Specifically, Costing 20 mins in single scale and 7 hours in multi-scale. Is it true?

**Comment by yazhe2017** [comment]
Yes, my test process is also very slow. Hope author could explain about it.

**Comment by junfu1115** [comment]
Due to evalute images with sliding the window, it would take a lot of time. Moreover, in multi-scale testing, we adopt several scales, and the largest scale is set to  2.2 times of original image, it might also take more time

**Comment by kaixinxiao** [comment]
@junfu1115 Thank you! But is it necessary to visualize the output using multi_scale? It costs dozens of hours but it has a better performance when I push it to cityscapes server. Do you apply multi_scale method about the performance of  81.5% on test set?

**Comment by xdsonglinliu** [comment]
Hi, I have a question.
Where can I see the sliding the window operation in the code when i test a large image(larger than the images used in training)? 
Anyone can help me? Thanks a lot!
@kaixinxiao 
@yazhe2017

**Comment by A-Kerim** [comment]
Indeed it is very slow; any solutions?
